### PR TITLE
CLI: change default 'nagios' behaviour to be consistent(HPC-8307)

### PR DESCRIPTION
**Pull request by wdpypere**
and some minor cleanups

### CODE REVIEW

File: lib/vsc/utils/script_tools.py [hunk 1]
```diff
@@ -281,6 +281,12 @@ def make_options(self, defaults=None):
 
         return ExtendedSimpleOption(options)
 
+    def ok(self, msg):
+        """
+        Convenience method that calls ExtendedSimpleOptions warning and exists with nagios OK exitcode
```
Comment (stdweird) [inline lib/vsc/utils/script_tools.py:286]: `warning`?
anyway, exitcode values should come from first elelent of https://github.com/hpcugent/vsc-utils/blob/master/lib/vsc/utils/nagios.py#L66

Comment (wdpypere) [reply lib/vsc/utils/script_tools.py]: @stdweird fixed the typo

**Comment by wdpypere** [comment]
@itkovian this is what I think should be the behaviour of CLI. Can you confirm we are on the same line? and if so, can you help met with test/script_tools.py. I'm confused again at what it is trying to test and why it fails. do I see it correctly the failing test should be moved to NrpeCLI?

**Comment by wdpypere** [comment]
@itkovian tests fixed

**Comment by itkovian** [comment]
LGTM, @stdweird you ok with this?

**Comment by stdweird** [comment]
looks ok. @wdpypere let me know when you can build and test this in production. you can ask anyone to merge it (but no new commits please)

**Pull request merged**